### DISCUSSION TITLE
OutOfMemoryError by clean install

### DISCUSSION THREAD

**Discussion by Dream-Work** [sulu/sulu#6634] [Q&A] [answered]
| Q | A
| --- | ---
| Bug? | maybe
| Sulu Version | sulu/skeleton (2.4.2)
| Browser Version | not yet in use :)

#### Actual Behavior

How does Sulu behave at the moment? 

Its  quit with an OutOfMemoryError

#### Expected Behavior

What is the behavior you expect?

Normal installation

#### Steps to Reproduce

Macbook Air M1 2020
Memory: 16GB

What are the steps to reproduce this bug? Please add code examples,
screenshots or links to GitHub repositories that reproduce the problem.

composer create-project sulu/skeleton sulu-ddev -n

`!!  14:39:34 CRITICAL  [php] Fatal Error: Allowed memory size of 134217728 bytes exhausted (tried to allocate 32768 bytes) ["exception" => Symfony\Component\ErrorHandler\Error\OutOfMemoryError^ { …}]
!!
!!  In 8c2d633b2f695d22dd4df9340ddd79a2.php line 721:
!!
!!    Error: Allowed memory size of 134217728 bytes exhausted (tried to allocate
!!    32768 bytes)
!!
!!
!!  cache:clear [--no-warmup] [--no-optional-warmers]
!!
!!
Script @auto-scripts was called via post-update-cmd
`

What can be done to make the installation successful?

thx
Sven
<img width="1415" alt="2022-06-02_16-47-50" src="https://user-images.githubusercontent.com/2921338/171656768-fa698cfc-82c5-47ab-8f8d-e454de4f8fb6.png">
<img width="1422" alt="2022-06-02_16-48-41" src="https://user-images.githubusercontent.com/2921338/171656902-ff6b6182-2ab8-4171-9d38-699cad40836f.png">

**Comment by alexander-schranz** [comment, marked as answer]
Looks like your instance has a `memory_limit` set in the CLI. Adopt your php.ini to set the `memory_limit` to a value you want but it should be larger then the default 128MB.

If its a development machine you can have a look at my PHP ini config: https://github.com/alexander-schranz/dotfiles/blob/master/php/php8.ini
